### DISCUSSION TITLE
ArcGIS Assistant - ArcGIS Enterprise portal & appid URL parameters?

### DISCUSSION THREAD

**Discussion by phoeffler84** [EsriPS/arcgis-assistant-feedback#43] [Q&A]
Apologies if I missed this, but can we pass the portal and appid parameters to the Assistant Beta through the URL, as we can with the ArcGIS Online Assistant? I looked through the documentation, tried a couple combinations modeled off of AGOA, and they didn't work.

We use this with a couple Portal deployments:
https://ago-assistant.esri.com/?portal=https://gis.fw.../portal&appid=5VuK...ajWkh
and when added to a web app in our galleries, it works well.

Thanks in advance

**Comment by bveldkamp** [comment]
Yes, the login experience is making me miserable every time. See also here #51

**Reply by frank-mancini** [reply]
+1
